notebooks/Analyse-picks.ipynb

```diff
@@ -6,7 +6,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import numpy as np"
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt\n",
+    "from sklearn.metrics.pairwise import euclidean_distances\n",
+    "from sklearn.neighbors import NearestNeighbors"
```

Comment:
let's create a conda environment to make sure the user can load those:

### first method
```
conda create -n cryotools python=3.8
conda activate cryotools
conda install jupyter scikit-learn numpy matplotlib
```

### second approach
create the following `environment.yml` file:
```
name: cryotools
channels:
  - defaults
dependencies:
  - python=3.8
  - jupyter
  - scikit-learn
  - numpy
  - matplotlib
```
then create the environment with:
`conda env create -f environment.yml`

Reply:
Fred do you mean to add
```
conda create -n cryotools python=3.8
conda activate cryotools
conda install jupyter scikit-learn numpy matplotlib
```
in the notebook?

Reply:
ah no, this is done outside the notebook. Are you familiar with Anaconda and conda environments?

These instructions should be in the README.md - make sure you can write a paragraph on those once you understand what it is useful for.

Reply:
<img width="1121" alt="Screen Shot 2020-08-04 at 14 41 03" src="https://user-images.githubusercontent.com/49101433/89347948-8f716800-d660-11ea-9a0e-c8a531c2d8e4.png">
Fred I wrote these in the README.md can you take a look at it and tell me if it's okay?

Reply:
Great! Now you can also commit the environment file so users can directly use it upon cloning